O365/message.py

```diff
@@ -755,7 +756,20 @@ def reply(self, to_all=True):
             return None
 
         message = response.json()
+        message_body_content_html  = message['body']['content']
 
+        pattern = re.compile(r'<b>Sent:</b> (.*?)<br>')
+
+        def change_sent_time(match):
+            sent_time_str = match.group(1)
+            sent_time = dt.datetime.strptime(sent_time_str, '%A, %B %d, %Y %I:%M:%S %p')
+            sent_time_new = sent_time.replace(tzinfo=dt.timezone.utc).astimezone(self.protocol.timezone)
```

Comment:
We should be more descriptive with this variable name, lets use `sent_time_as_timezone`. This will make the line too long past the 120char limit, might need to do something like:
```
            sent_time_as_timezone = (sent_time.replace(tzinfo=dt.timezone.utc)
                                     .astimezone(self.protocol.timezone))
```